digdag-core/src/main/java/io/digdag/core/workflow/WorkflowExecutor.java

```diff
@@ -342,6 +342,18 @@ private void noticeStatusPropagate()
         }
     }
 
+    public void noticeRunWhileConditionChange()
+    {
+        propagatorLock.lock();
+        try {
+            // don't set propagatorNotice but break wait at runWhile
```

Comment:
👍 I asked people around me to know how in and at are different...and still not 100% sure about the difference....